config/resources/domain.json

```diff
@@ -590,12 +624,44 @@
     },
     "central-worship-services": {
       "class": "ere:CentraalBestuurVanDeEredienst",
-      "super": ["worship-administrative-units"],
+      "super": [
+        "worship-administrative-units"
+      ],
       "new-resource-base": "http://data.lblod.info/id/centraleBesturenVanDeEredienst/"
     },
+    "kbo-administrative-units": {
+      "class": "ext:KboEenheid",
+      "super": [
+        "organizations"
+      ],
+      "attributes": {
+        "rechtsvorm": {
+          "type": "string",
+          "predicate": "ext:rechtsvorm"
+        },
+        "short-name": {
+          "type": "string",
+          "predicate": "ext:shortName"
+        },
```

Comment:
**Issue:** In the feature passport: https://app.gitbook.com/o/-MP9Yduzf5xu7wIebqPG/s/-M_5cpNQ4inMjiC5hhyJ/update-mechanisms/show-kbo-data-and-compare-with-op-data#data-mapping

- `short-name` should be stored in `<http://www.w3.org/2004/02/skos/core#altLabel>`
- `rechtsvorm` in `<http://www.w3.org/ns/regorg#legalName>`

Reply:
@aliokan I do not think the `<http://www.w3.org/ns/regorg#legalName>` predicate should be used for `rechtsvorm`. This predicate is used in OSLO for "wettelijke naam". See the [FP](https://abb-vlaanderen.gitbook.io/doc-organisatieportaal/data-quality/organisation-names#oslo-standardl) for OP-2999 that will introduce the legal name for organisations. Whereas the `rechtsvorm` resembles more our types of classifications, although the available options differ.

Reply:
i have changed these according to discussion of last daily